public/script.js

```diff
@@ -101,46 +96,36 @@ function makeBarChart(id, title, csv, annotations, showGroups) {
                 yAxes: [{
                     stacked: true,
                     ticks: {
-                        display: showGroups,
+                        display: showGroups
                     }
                 }],
                 xAxes: [{
-                    id: 'x',
+                    id: "x",
                     stacked: false,
                     ticks: {
                         min: 0,
                         max: 100,
-                        callback: function(value) {
-                            return value + "%"
-                        }
+                        callback: value => value + "%"
                     }
-                }],
+                }]
             },
             annotation: {
                 drawTime: "afterDatasetsDraw",
-                annotations: annotations,
-            },
+                annotations: annotations
+            }
         }
     });
 }
 
 async function initializeLineCharts(latestDataDate) {
-    let csv = await d3.csv("line.csv");
-    let herdImmunityDate = '';
-    csv.forEach(function(row) {
-        if (herdImmunityDate != '' || row.dose != "2_wait") {
-            return;
-        }
-        let vaccinated = parseInt(row.vaccinated);
-        let population = parseFloat(row.population);
-        if ((vaccinated / population) > 0.7) {
-            herdImmunityDate = row.real_date;
-        }
-    });
+    const csv = await d3.csv("line.csv")
+    const herdImmunityDate = csv.find(row => {
+        const vaccinated = parseInt(row.vaccinated);
+        const population = parseFloat(row.population);
+        return row.dose !== "2_wait" && (vaccinated / population) > 0.7;
```

Comment:
Should be `row.dose === "2_wait"` - we want the first 2nd dose+7d that is on >70% of the population.